components/elm/bld/namelist_files/namelist_defaults.xml

```diff
@@ -274,10 +274,13 @@ ic_tod="0" sim_year="2000" glc_nec="0" use_crop=".true." irrigate=".true." nu_co
 <!-- Initial conditions for DYAMOND1, DYAMOND2 and for seasons of ENSO neutral year 2013 -->
 <finidat hgrid="ne1024np4.pg2" mask="ICOS10" ic_ymd="20160801">lnd/clm2/initdata/20220928.I2010CRUELM.ne1024pg2_ICOS10.elm.r.2016-08-01-00000.nc</finidat>
 <finidat hgrid="ne1024np4.pg2" mask="ICOS10" ic_ymd="20200120">lnd/clm2/initdata/20221218.F2010-CICE.ne30pg2_ne1024pg2.elm.r.2020-01-20-00000.nc</finidat>
+<finidat hgrid="ne1024np4.pg2" mask="ICOS10" ic_ymd="19941001">lnd/clm2/initdata/20231226.I2010CRUELM.ne1024pg2_ICOS10.elm.r.1994-10-01-00000.nc</finidat>
```

Comment:
@brhillman , looks like you made these changes to ELM too. Can you give a quick explanation? The commit was:

```
commit f8e9303e7fcc7867883300b52df3c0bd44f7f593
Author: Benjamin Hillman <bhillma@sandia.gov>
Date:   Thu Feb 22 13:51:09 2024 -0500

    Fix finidat for elm ne1024
```

Reply:
Additional inputfiles for different start dates for decadal runs. Only affect ne1024 and ne256, I don't see a problem here.